pace_neutrons/__init__.py

```diff
@@ -57,7 +66,7 @@ def __init__(self, matlab_path: Optional[str] = None, matlab_version: Optional[s
                 pass
         if not initialized:
             raise RuntimeError(
-                f"No MATLAB versions found. Please use: [{', '.join([version['version'] for version in _VERSIONS])}]\n "
+                f"No MATLAB versions found. Please use: [{', '.join([version['version'] for version in _VERSIONS])}] (https://uk.mathworks.com/products/compiler/matlab-runtime.html)\n "
```

Comment:
Please could you explain this suggestion? It appears [ ] have been removed. I just want to try and understand what difference this will make to the code and if the output will differ at all as well as why it may have originally been done using [ ]

Reply:
It avoids the unnecessary construction of a `list`. `join` accepts generators so we don't need to build a `list` and `join` it we can just `join` the generator

Reply:
ok thanks for this. I am just going through and making these changes now so I should have a new commit up before lunch